image.php

```diff
@@ -74,7 +79,28 @@
 	exit;
 }
 
+// we must find a content type of image
+$found_img = false;
+$forwarded_headers = [];
 foreach (explode("\r\n", $headers) as $header) {
+	if (strpos($header, "Content-Type: image/") === 0) {
+		$found_img = true;
+		$forwarded_headers[] = $header;
+	}
+	
+	if (strpos($header, "Content-Length: ") === 0) {
```

Comment:
I don't recommend passing this through, but you can send it based on the actual bytes in the image.